docs/Configuration.md

```diff
@@ -0,0 +1,40 @@
+# Configuration
+
+* [Receivers](#receivers)
+* [Processors](#processors)
+* [Exporters](#exporters)
+
+## Receivers
+
+### Filelog Receiver
+
+Filelog Receiver simple config
+
+Link to [Filelog Receiver detailed configuration][filelogreceiver_readme]
+
+### Syslog Receiver
+
+Syslog Receiver simple config
+
+Link to [Syslog Receiver detailed configuration][syslogreceiver_readme]
+
+### Telegraf Receiver
+
+Telegraf Receiver simple config
+
+Link to [Telegraf Receiver detailed configuration][telegrafreceiver_readme]
+
+### OTLP Receiver
+
+OTLP Receiver simple config
+
+Link to [OTLP Receiver detailed configuration][otlpreceiver_readme]
+
+## Processors
```

Comment:
What about processors :thinking:

Reply:
If we include any we should provide info how to configure them. It's a stub now.